tests/test_tools.py

```diff
@@ -15,13 +15,15 @@
 import unittest
 from pathlib import Path
 from typing import Dict, Optional, Union
+from unittest.mock import patch, MagicMock
 
+import mcp
```

Comment:
You test needs `mcp` and I think it is not installed.

Reply:
it's a dependency of mcpadapt

Reply:
so should be fine